resources/styles/components/course-calendar/add.less

```diff
@@ -7,4 +7,33 @@
     display: block;
     padding: 3px 20px;
   }
+
+  .x-add-plan(@plan-color) {
+    color: @tutor-neutral-darker;
+    border-left: 5px solid @white;
+    background: none;
+    transition: background .1s ease-in, border .1s ease-in;
+
+    &:hover,
+    &.active,
+    &.open {
+    	border-left: 5px solid @plan-color;
+      background: fade(@plan-color, 20%);
+      transition: background .1s ease-in, border .1s ease-in;
+    }
+  }
+
+  li {
+    a[href*="readings"] { .x-add-plan(@reading-color); }
```

Comment:
seems like we could provide a `[data-assignment-type='reading']` attribute on all the elements to make this easier/consistent